tests/testthat/test-equal.R

```diff
@@ -221,6 +221,13 @@ test_that("equal_scalar() compares", {
   expect_error(test_equal_scalar(expression(0), 0, expression(1), 1), "Unsupported")
 })
 
+test_that("vec_equal() silently falls back to base data frame", {
+  expect_silent(expect_identical(
+    vec_equal(foobar(mtcars), foobar(tibble::as_tibble(mtcars))),
```

Comment:
What happens with df-cols where one df-col is a data frame and one is a tibble? i.e. is this recursive?

Reply:
Yes the fallback parameter is passed recursively. This parameter passing isn't implemented yet when the ptype2 method is implemented at R level. Once we pass it, this should work if the method passes the `...` back to `df_ptype2()` as we have documented.

Reply:
I think we should wait for the next release cycle before passing extra parameters to ptype2 methods.